src/functions/template-tags/html.php

```diff
@@ -178,31 +179,36 @@ function tribe_format_field_dependency( $deps ) {
 
 		// Handle the ID component.
 		if ( 'id' === $attr ) {
-			$dependency .= " data-depends=\"{$value}\"";
+			$dependency .= " data-depends=\"#{$value}\"";
+			continue;
+		}
+
+		// Handle the dependent parent component.
+		if ( 'parent' === $attr ) {
+			$dependency .= " data-dependent-parent=\"{$value}\"";
 			continue;
 		}
 
 		// Handle boolean values.
 		if ( is_bool( $value ) ) {
 			if ( $value ) {
-				$dependency .= " data-{$attr}";
+				$dependency .= " data-condition-{$attr}";
```

Comment:
Could you explain this? It looks to me like the logic is changing here.

Reply:
This code did not work correctly as it was. It checks for `data-condition-foo` but we were setting `data-foo`

Reply:
History: this was created and then _not used_ for the list widget.

Reply:
I want to say we did something with this for ET tickets in the classic editor, let me go check and see if we had built something extra based on that.

Reply:
Looks clear, just wanted to check!